### PR TITLE
Downstream flow build variables

### PR DESCRIPTION
**Pull request by jglick**
[JENKINS-29169](https://issues.jenkins-ci.org/browse/JENKINS-29169)

@reviewbybees

### CODE REVIEW

File: support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java [hunk 1]
```diff
@@ -160,7 +161,12 @@ public String getId() throws AbortException {
         if (build instanceof AbstractBuild) {
             return Collections.unmodifiableMap(((AbstractBuild<?,?>) build).getBuildVariables());
         } else { // not applicable
```
Comment (amuniz) [inline support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java]: Seems it is applicable now. Remove this comment?

**Comment by amuniz** [comment]
:bee:

File: support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java (hunk 1 above)
Comment (jglick) [inline support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java]: Well, `AbstractBuild.getBuildVariables` is inapplicable, but I agree the comment could be confusing now.

**Comment by amuniz** [comment]
:bee:

**Pull request merged**